docs/development/adding-a-package-manager.md

```diff
@@ -14,11 +14,23 @@ Common application logic for Renovate, not specific to particular managers, usua
 
 The manager's `index.ts` file supports the following values/functions:
 
-- extractPackageFile
-- extractAllPackageFiles
-- getRangeStrategy (optional)
-- language (optional)
-- supportsLockFileMaintenance (optional)
+| Value/function                | Optional |
```

Comment:
Do we want a column for the `async` property? Then we list all the properties in the table.

Reply:
🤔  makes sense